aws/logs_monitoring/tools/test_harness/recorder/recorder.py

```diff
@@ -12,7 +12,7 @@
 
 class RecorderHandler(BaseHTTPRequestHandler):
     def __init__(self, request, client_address, server):
-        super().__init__(request, client_address, server)()
+        super().__init__(request, client_address, server)
```

Comment:
This is not valid and was raising exceptions in the integration tests, so I cleaned it up